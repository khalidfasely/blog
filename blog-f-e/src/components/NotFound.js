import React from "react";
import { BrowserRouter, Link } from "react-router-dom";

const NotFound = () => (
    <div data-testid='404-message'>
        404 - Not Found <BrowserRouter><Link to='/'>Go Home</Link></BrowserRouter>
    </div>
);

export default NotFound;