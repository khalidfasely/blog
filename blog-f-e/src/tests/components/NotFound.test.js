import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../components/NotFound";

test('Should render not found page correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});