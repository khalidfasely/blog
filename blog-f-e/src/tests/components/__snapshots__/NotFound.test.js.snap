// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render not found component correctly 1`] = `
<DocumentFragment>
  <div
    data-testid="404-message"
  >
    404 - Not Found 
    <a
      href="/"
    >
      Go Home
    </a>
  </div>
</DocumentFragment>
`;
